### PR TITLE
Makes crayons not racist anymore

### PR DESCRIPTION
**Pull request by Niknakflak**
image
- Adds the black crayon to the game and the crayon box
- Used to dye things black and make really really dark graffiti
- Krav_maga is repathed for the reason stated below after the latest commit

### CODE REVIEW

**Comment by Iamgoofball** [comment]
>>>washing machine code

**Comment by Niknakflak** [comment]
I know right...

**Comment by Fox-McCloud** [comment]
Should make a white crayon as well. I know, someone is going to mention that mime's crayon, but that's technically a black+white crayon in one (that can flip between the two).

**Comment by Niknakflak** [comment]
There is a white crayon that isn't the mime crayon

**Comment by PKPenguin321** [comment]
blm

**Comment by Fox-McCloud** [comment]
>There already is a white crayon that isn't the mime crayon

Thanks. I'm a derp.

**Comment by Zilenan91** [comment]
If you take the black crayon out of the box can you put it back in? I think that was an issue a while ago where they spawned inside the box but wouldn't be able to be put back in.

**Comment by Niknakflak** [comment]
yes

**Comment by optimumtact** [comment]
Finish soon please

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -144,7 +144,7 @@
 		. = TRUE
 
 /obj/item/toy/crayon/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = 0, datum/tgui/master_ui = null, datum/ui_state/state = hands_state)
-	// god bless tgui and all of its arguments
+	// tgui is a plague upon this codebase
```
Comment (ChangelingRain) [inline code/game/objects/items/crayons.dm]: spaces directly after // is also a plague

**Comment by Niknakflak** [comment]
I dont know how

**Comment by Supermichael777** [comment]
black gloves probably have the wrong sprite name

**Comment by Niknakflak** [comment]
I tried that

**Comment by Niknakflak** [comment]
This is fully working and functional now.  Things that have changed
-krav_maga was repathed because washing machine code was seeing it as a child and turning the black crayon dyed gloves into it.  This only happened for the black gloves because the other gloves dont have a different sprite for their children when it comes to colored gloves.  Aka: Quality washing machine code, but the path was dumb anyway.  They were given fire resistance so it wouldn't be a "nerf" when they were changed away from children of black gloves.
-black gloves item state was changed from bgloves to blackgloves to be consistent with the other gloves
-Dying gloves black now works.  HOORAY THIS ONLY TOOK LIKE AN HOUR TO FIGURE OUT WHY

Ready for merge!

**Pull request merged**